scripts/sysman/EXT_Exp_power_domain.rst

```diff
@@ -0,0 +1,26 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_power_domain_property:
+
```

Comment:
The name here does not match the actual extension name, please change to match, as follows:

```suggestion
.. _ZES_extension_power_domain_properties:
```

Reply:
thanks @wdamon-intel applied your suggestion automatically